docs/serving/load-balancing/activator-capacity.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Configuring Activator capacity"
+linkTitle: "Configuring Activator capacity"
+weight: 50
+type: "docs"
+---
+
+# Configuring Activator capacity
+
+If there's more than one Activator in the system, Knative will put as many of them on the request path as it thinks is necessary (if target burst capacity is not set to 0). If available, it'll pick at least two for high availability reasons. The actual number of Activators is calculated via a given _Activator capacity_ like such: `(replicas * target + targetBurstCapacity)/activatorCapacity`. That means, there are enough Activators in the routing path to handle the theoretic capacity of the existing application, including any additional target burst capacity.
```

Comment:
How about something (for the first sentence) like: "..Knative will put as many of them on to the request path as it thinks is necessary to handle the current request load plus the Target Burst Capacity. If Target Burst Capacity is 0 Knative only puts the activator in to the path when the revision is scaled to zero"

Reply:
Also because I happen to know the style guide will not let us use parentheticals :D